queries.yaml

```diff
@@ -1303,6 +1317,22 @@ post_process:
         (kind == 'ferry' and zoom < 14) or
         (kind == 'rail' and zoom < 15)
 
+  # drop ref, relation, and odd name tags till the end
+  # these aren't used for rendering and bloat property size
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 17
+      properties:
+        - ref
+        - osm_relation
+        - alt_name
+        - alt_name:ar
+        - alt_name:en
```

Comment:
This can be verified in 512 px tile 14-9612-6755, mostly in minor_roads but also in major_roads.

For example:  https://www.openstreetmap.org/way/24986420

<img width="1199" alt="image" src="https://user-images.githubusercontent.com/853051/153135490-934267c0-0353-462d-8224-31c57e239219.png">

Reply:
Change looks good with latest build, dropping these alt_names